gradle.properties

```diff
@@ -1,5 +1,5 @@
 kotlin.code.style=official
 kotlin_version=1.4.10
-release_version=0.2.0
+release_version=0.2.1
```

Comment:
Please update minor version since this PR extends functionality (also don't forget to update versions in README too)